draft-ietf-intarea-provisioning-domains.md

```diff
@@ -713,31 +746,26 @@ document can be used:
 
 ~~~
 {
-  "identifier": "cafe.example.com",
-  "expires": "2017-07-23T06:00:00Z",
+  "identifier": "cafe.example.com.",
+  "expires": "2020-05-23T06:00:00Z",
   "prefixes": ["2001:db8:1::/48", "2001:db8:4::/48"],
 }
 
 {
-  "identifier": "company.foo.example.com",
-  "expires": "2017-07-23T06:00:00Z",
+  "identifier": "company.foo.example.com.",
+  "expires": "2020-05-23T06:00:00Z",
   "prefixes": ["2001:db8:1::/48", "2001:db8:4::/48"],
   "vendor-foo":
-  	{
-  		"private-key": "private-value",
-  	},
+    {
+        "private-key": "private-value",
```

Comment:
2nd thought here... "private-key" could be mistaken as 'private/public key pair'

Reply:
I'd prefer to leave as is, since it didn't come up (I was just fixing tabs)

Reply:
Do not look for trouble ;-)